api/v1/openapi.yaml

```diff
@@ -974,11 +974,25 @@ definitions:
         type: array
         items:
           type: string
+      ingress-source-rule-labels:
+        description: Labels of ingress L3 rules in effect on this endpoint
```

Comment:
Is it feasible to do something like this instead?

    "l4": {
        "egress": [{
             "rule": "{\n  \"port\": 80,\n  \"protocol ...",
             "derived-from-rules": [{["foo", "bar"]},{["blah"]}]
        },{
            "rule": "{\n  \"port\": 80,\n  \"protocol ...",
            "derived-from-rules": [{["baz2"]}]
        }]
    }

Reply:
@raybejjani also, what's the output for `policy.allowed-consumers`?

Reply:
I don't think I changed it. A random endpoint outputs:
```json
    "policy": {
      "allowed-consumers": [
        1
      ],
```

Reply:
I'll merge the derived-from thing with rule. Are we ok changing the API this way? Do I need to roll a version somewhere?

Reply:
> I'll merge the derived-from thing with rule. Are we ok changing the API this way? Do I need to roll a version somewhere?

No reason to bump pre 1.0